tests/airflow/test_graph.py

```diff
@@ -29,6 +31,7 @@
 from cosmos.profiles import PostgresUserPasswordProfileMapping
 
 SAMPLE_PROJ_PATH = Path("/home/user/path/dbt-proj/")
+SOURCE_RENDERING_BEHAVIOR_ENV = os.getenv("SOURCE_RENDERING_BEHAVIOR", "none")
```

Comment:
May need to type cast?
```suggestion
SOURCE_RENDERING_BEHAVIOR_ENV = SourceRenderingBehavior(os.getenv("SOURCE_RENDERING_BEHAVIOR", "none"))
```